Client/Application/AppDelegate.swift

```diff
@@ -37,6 +37,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             // Use a clean profile for each test session.
             profile = BrowserProfile(localName: "testProfile", app: application)
             profile.files.removeFilesInDirectory()
+            profile.prefs.clearAll()
```

Comment:
IMO, it makes sense here since we already have the related `profile.files.removeFilesInDirectory()` right above it. I think we'll also need to add some test-specific prefs here, such as bypassing the first run UI. Those prefs can't be in `XCTestCase` since the app runs parallel to the tests, and the startup is executed before the test has started running.